Bottom: 529ccd18fc6dc7da3da219c376cda494cb17bcf8
Top:    39d11e62a9a24176be2d05efb5f6becc0c8e226b
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-08-29 15:16:33 +0800

Refresh of vhost_net-tx-batching

---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index 40f23b4..20614eb 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -505,10 +505,10 @@ static void handle_tx(struct vhost_net *net)
 		}
 
 		for (i = 0; i < avails; i++) {
-			head = vhost_get_vq_desc2(vq, vq->iov,
-						  ARRAY_SIZE(vq->iov),
-						  &out, &in, NULL, NULL,
-						  indices[i]);
+			head = __vhost_get_vq_desc(vq, vq->iov,
+						   ARRAY_SIZE(vq->iov),
+						   &out, &in, NULL, NULL,
+						vhost16_to_cpu(vq, indices[i]));
 			if (in) {
 				vq_err(vq, "Unexpected descriptor format for TX: "
 				       "out %d, int %d\n", out, in);
